src/libs/antares/study/parts/common/series.h

```diff
@@ -33,6 +33,54 @@
 
 namespace Antares::Data
 {
+
+class TimeSeries
```

Comment:
We should take the opportunity of this refactoring to have some unit tests on those classes, in particular this one.

We need to test the indexing logic and the serialization methods.